sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_daily_v1/query.sql

```diff
@@ -30,20 +30,13 @@ WITH fxa_events AS (
     AND fxa_log IN ('content', 'auth', 'oauth')
     -- re-using the filter from users_services_daily_v1 for consistency across the models
     -- at some point in the future we should re-evaluate this list
```

Comment:
This PR looks like we're re-evaluating this list, so how about we remove this comment?
We can maybe add a comment saying that three fxa_activity events are enough to detect user's activity.

Reply:
I missed that, thanks for pointing it out.